packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -128,6 +128,20 @@ export interface ContainerDefinitionOptions {
    */
   readonly secrets?: { [key: string]: Secret };
 
+  /**
+   * Time duration to wait before giving up on resolving dependencies for a container.
```

Comment:
The time duration `in seconds` might be more descriptive.

Reply:
not necessary I think. If they put `cdk.Duration.millis(2000)` should also work.

Reply:
yes, but as a user if I put 2200 millis, I expect 2.2 seconds, not 2 seconds.

Reply:
The reason why using cdk Duration I think is because from users' perspective they don't need to care the time format using in CFN (in minutes or seconds or millis).